Source/WebCore/testing/Internals.idl

```diff
@@ -1311,6 +1326,7 @@ enum ContentsFormat {
 
     undefined setCookie(CookieData cookieData);
     sequence<CookieData> getCookies();
+    sequence<WebDriverCookieData> webDriverGetCookies(Document document);
```

Comment:
Is there any reason we can't just use `CookieData` and the above functions? Seems like duplicate logic.